spec/acceptance/reporting/event_spec.rb

```diff
@@ -8,49 +8,51 @@
       instance: servicenow_instance.uri,
       user: servicenow_config['user'],
       password: "Sensitive('#{servicenow_config['password']}')",
-      skip_certificate_validation: Helpers.skip_cert_check?,
+      skip_certificate_validation: true,
+      pe_console_cert_validation: 'none',
     }
   end
 
   let(:setup_manifest) do
     to_manifest(declare('Service', 'pe-puppetserver'), declare('class', 'servicenow_reporting_integration::event_management', params))
   end
 
+  let(:fqdn) do
+    server.run_shell('facter fqdn').stdout.chomp
+  end
+
   include_context 'event query setup'
 
   it 'sends a node_report event' do
     set_sitepp_content(declare('notify', 'foo'))
     trigger_puppet_run(server, acceptable_exit_codes: [2])
     event = Helpers.get_single_record('em_event', query)
     additional_info = JSON.parse(event['additional_info'])
-
     expect(event['source']).to eql('Puppet')
     expect(event['type']).to eql('node_report_intentional_changes')
     expect(event['severity']).to eql('5')
     expect(event['message_key']).not_to be_empty
     expect(event['node']).not_to be_empty
-    expect(event['event_class']).to match(Regexp.new(Regexp.escape(server.uri)))
-    expect(event['description']).to match(Regexp.new(Regexp.escape(server.uri)))
+    expect(event['event_class']).to match(Regexp.new(Regexp.escape(fqdn)))
```

Comment:
This whole function definition `Regexp.new(Regexp.escape(fqdn))` could be moved into the fqdn let could it not?